src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -0,0 +1,9 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class ColorSupplier {
+    public String getRandomColor() {
```

Comment:
It's better to use an instance of `Random` instead of creating a new one every time. Consider creating a private final field and using it in the `getRandomColor()` method. 🌈